core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/SnapshotApiServiceSpec.scala

```diff
@@ -154,4 +154,45 @@ class SnapshotApiServiceSpec extends ApiServiceSpec {
       sealRoute(services.snapshotRoutes) ~>
       check { assertResult(StatusCodes.NotFound) {status} }
   }
+
+  it should "return 200 when a user lists all workspaces" in withTestDataApiServicesAndUser(testData.userReader.userEmail.value) { services =>
```

Comment:
nit: test name should be something like "... when a user lists snapshots in a workspace"

Reply:
Done!